entries/g/gwu.edu.json

```diff
@@ -0,0 +1,21 @@
+{
+  "George Washington University": {
+    "domain": "gwu.edu",
```

Comment:
I can't find a DNS record for gwu.edu :thinking:

Reply:
When will people start adding root domain DNS records?
Thank you for catching that.

Reply:
I guess they don't want to assume people use HTTP and not [Gopher](https://youtu.be/b4aApVkvrNU)?